### PR TITLE
Gitpodify the Signoz

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
As we all know [`signoz/signoz`](https://github.com/SigNoz/signoz) is taking part in GitHub Externship Program. For Me as a beginner, the major block is to set up the project locally.

So, Instead of configuring a local setup, we can use [Gitpod](https://www.gitpod.io/), an automated cloud dev environment.

After the merging of this PR, you can directly run this repo. in your browser. It is will be ready to run, build, test & ship 🚀  the new features.

[Detailed Docs about the Gitpod](https://github.com/Siddhant-K-code/signoz/blob/000d1226f9842694ecea8370d8e8d4520ea98e86/CONTRIBUTING.md#L68)

> ⭐  This Will Help New Contributors to start building & setting up easily.
---
### Test
I have Tested it end-to-end it is building & running perfectly on Gitpod.

You Can Test through here, is working upon `siddhant-k-code/signoz`

[Open in Gitpod](https://gitpod.io/#https://github.com/Siddhant-K-code/signoz)

### CODE REVIEW

File: frontend/src/constants/env.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 export const ENVIRONMENT = {
-	baseURL: process?.env?.FRONTEND_API_ENDPOINT || '',
+	baseURL: process?.env?.FRONTEND_API_ENDPOINT || process.env.GITPOD_WORKSPACE_URL.replace('://','://8080-') ||'http://localhost:8080',
```
Comment (palash-signoz) [inline frontend/src/constants/env.ts:2]: can you please add ""

**Review by palash-signoz — APPROVED** [review]

File: frontend/src/constants/env.ts (hunk 1 above)
Comment (Siddhant-K-code) [reply frontend/src/constants/env.ts]: Done, Resolved.

**Review by ankitnayan — CHANGES_REQUESTED** [review]

File: frontend/webpack.config.js [hunk 2]
```diff
@@ -41,6 +41,7 @@ const config = {
 			publicPath: '/',
 			watch: true,
 		},
+		allowedHosts: 'all',
```
[the remark this answers is not in the document]
Comment (Siddhant-K-code) [reply frontend/webpack.config.js:44]: It is not for production. It is just for the development mode. 


https://github.com/SigNoz/signoz/blob/af9b85f0aa5f2ab72f664306f9753fb78980926c/frontend/webpack.config.js#L30

[the remark this answers is not in the document]
Comment (palash-signoz) [reply frontend/webpack.config.js:44]: @Siddhant-K-code can you make this one in dev config ?

[the remark this answers is not in the document]
Comment (Siddhant-K-code) [reply frontend/webpack.config.js:44]: > can you make this one in dev config ?

what you wanted to config?

**Review by ankitnayan — APPROVED** [review]

**Comment by Siddhant-K-code** [comment]
Thanks for Merging It 🚀

**Pull request merged**